mix.exs

```diff
@@ -48,7 +48,7 @@ defmodule BencheeHTML.Mixfile do
   defp deps do
     [
       {:benchee, github: "PragTob/benchee"},
-      {:benchee_json, github: "PragTob/benchee_json"},
```

Comment:
I'll be holding off on reviewing this until we talked about the structure in the JSON formatter